src/main/resources/application.yml

```diff
@@ -21,4 +21,7 @@ spring:
       org.hibernate: INFO
 server:
   servlet:
-    context-path: /bank
\ No newline at end of file
+    context-path: /bank
+age:
```

Comment:
to wlasnie to o czym mowilem. Lepiej jest zrobic to w jednej linijce bo ogolnie struktura powinna byc taka ze kazdy poziom pliku yml to powinna byc jedna grupa 
Np masz 
credentials: 
   dbLogin: user
   dbPassword: somethingElse
   adminPassword: blablabla 
  I tu masz grupowanie po kluczu credentials. Dzielenie tego w taki sposob na poziomy nie ma sensu. 
  Sprawdz czy mozna uzywac - w nazwach kluczy. 
Bo moze da sie zrobic age-of-majority :D

Reply:
Jeżeli chodzi o samą aplikację, to działa w różnym zapisie, ostatecznie zrobiłem age-of-majority.
Niestety niezależnie od zapisu klasa z testami nie widzi tego ustawienia. Szukałem, ale nie trafiłem na nic działającego. Doszukałem się informacji, że adnotacje powinny działać w pliku .properties, a nie działają w .yml. 
Także na tą chwilę wstrzymuję dociekania do kolejnego commita.